### ISSUE TITLE
Helm install of 50 nodes or less version?

### ISSUE THREAD

**Issue by KlavsKlavsen** [projectcalico/calico#5585]
How can I install the "50 nodes or less" version of calico using Helm chart? - to avoid the overhead on small k8s setups? its quite a lot more pods for typha version :(
You still support this for manual installation: https://projectcalico.docs.tigera.io/getting-started/kubernetes/self-managed-onprem/onpremises

But I see no values that allows me to switch to that installation type?

**Comment by caseydavenport** [comment]
Yep, that doc is outdated and I'm working on improvements to it. The "50 nodes or less" doens't mean you shouldn't install the greater than 50 nodes option with Typha - on small clusters, Typha uses almost no resources, so it's not something to be concerned about. The "less than 50 nodes" option is really a legacy option that only works with manfiests.
